src/platform/site-wide/announcements/containers/Announcement.jsx

```diff
@@ -12,10 +12,14 @@ class Announcement extends React.Component {
   }
   dismiss = () => {
     const {
-      announcement: { name: announcementName, relatedAnnouncements = [] },
+      announcement: {
+        name: announcementName,
+        showEverytime = false,
+        relatedAnnouncements = [],
+      },
     } = this.props;
 
-    this.props.dismissAnnouncement(announcementName);
+    this.props.dismissAnnouncement(announcementName, showEverytime);
```

Comment:
Announcements were designed to be dismissed once, and that dismissal to persist via localstorage. In this case, however, we're showing the dialog every time, so I had to add a `showEverytime` flag.